src/Resources/public/shop/setono-callout-plugin.css

```diff
@@ -0,0 +1,5 @@
+.setono-callout { }
```

Comment:
Doesn't look like something we need?

Reply:
Yeah, it not included to code (injection commented), but it probably will be needed in future. Will remove for now.

Reply:
Cool